docs/source/index.rst

```diff
@@ -1,459 +1 @@
-    :alt: Slack-Swung
-    :target: https://swung.slack.com/messages/gstools
-.. |Gitter-GSTools| image:: https://img.shields.io/badge/Gitter-GeoStat--Framework-ed1965?logo=gitter&style=flat
-    :alt: Gitter-GSTools
-    :target: https://gitter.im/GeoStat-Framework/GSTools
-.. |Email| image:: https://img.shields.io/badge/Email-GeoStat--Framework-468a88?style=flat&logo=data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbDpzcGFjZT0icHJlc2VydmUiIHdpZHRoPSI1MDAiIGhlaWdodD0iNTAwIj48cGF0aCBkPSJNNDQ4IDg4SDUyYy0yNyAwLTQ5IDIyLTQ5IDQ5djIyNmMwIDI3IDIyIDQ5IDQ5IDQ5aDM5NmMyNyAwIDQ5LTIyIDQ5LTQ5VjEzN2MwLTI3LTIyLTQ5LTQ5LTQ5em0xNiA0OXYyMjZsLTIgNy0xMTUtMTE2IDExNy0xMTd6TTM2IDM2M1YxMzdsMTE3IDExN0wzOCAzNzBsLTItN3ptMjE5LTYzYy0zIDMtNyAzLTEwIDBMNjYgMTIxaDM2OHptLTc5LTIzIDQ2IDQ2YTM5IDM5IDAgMCAwIDU2IDBsNDYtNDYgMTAxIDEwMkg3NXoiIHN0eWxlPSJmaWxsOiNmNWY1ZjU7ZmlsbC1vcGFjaXR5OjEiLz48L3N2Zz4=
-    :alt: Email
-    :target: mailto:info@geostat-framework.org
-.. |Twitter| image:: https://img.shields.io/twitter/follow/GSFramework?style=social
-    :alt: Twitter Follow
-    :target: https://twitter.com/GSFramework
+.. mdinclude:: ../../README.md
```

Comment:
Woah! What kind of black magic is this?!?

Reply:
We used this for the changelog for ages.